sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -1,29 +1,28 @@
 <template>
-  <v-card>
-    <v-container :fluid="true">
-      <v-row class="m-2">
-        <v-col class="category-header-large color-blue">{{ title }}</v-col>
-        <v-col>
-          <check-permission-role :role="allowedRole">
-            <template #="{ notAllowed }">
-              <v-btn
-                @click="addNewNote"
-                class="float-right"
-                color="primary"
-                prepend-icon="fa:far fa-edit"
-                :disabled="notAllowed"
-              >
-                Create new note
-              </v-btn>
-            </template>
-          </check-permission-role>
-        </v-col>
-      </v-row>
-      <v-row class="m-2" v-if="!notes || notes.length === 0"
-        ><v-col
-          >No notes found. Please click on create new note to add one.</v-col
-        ></v-row
-      >
+  <v-container :fluid="true">
```

Comment:
our plan was to add the new template to all tab content right?

Reply:
Yes, fixed it.